src/solver/ts-generator/thermal.cpp

```diff
@@ -45,7 +46,6 @@ class GeneratorTempData final
 {
 public:
     GeneratorTempData(Data::Study& study,
-                      Solver::Progression::Task& progr,
```

Comment:
Can't progression be given in all cases ? Why remove it ?

Reply:
I thought it was only used in the GUI
only add logs like this in the solver
`[solver][progress] task 0 thermal, year: 0, 100`